composites/Plugin/ContentAnalysis/reducers/contentAnalysis/overallScoreReducer.js

```diff
@@ -0,0 +1,39 @@
+import { SET_OVERALL_SCORE_READABILITY, SET_OVERALL_SCORE_SEO } from "../../actions/contentAnalysis";
+
+/**
+ * Initial state
+ */
+const initialState = {};
+
+/**
+ * Sets the overall score for SEO results for one or more keywords.
+ *
+ * @param {Object} action The action.
+ * @returns {Object} The overall score per keyword.
+ */
+function setOverallScoreSeo( action ) {
+	let scorePerKeyword = {};
+	action.scorePerKeyword.forEach( function( keywordResultsPair ) {
```

Comment:
I'd probably call `setOverallScoreSeo( action.scorePerKeyword )` elsewhere, rather than making this function work on action objects. Small difference though.